encoding/gocode/gen.go

```diff
@@ -75,10 +75,14 @@ func GenerateTypesOpenAPI(sch thema.Schema, cfg *TypeConfigOpenAPI) ([]byte, err
 	if cfg.NoOptionalPointers {
 		depointer = depointerizer(true)
 	}
-	cfg.ApplyFuncs = append(cfg.ApplyFuncs, depointer, fixRawData())
 
+	applyFuncs := []dstutil.ApplyFunc{depointer, fixRawData()}
 	if !cfg.UseGoDeclInComments {
-		cfg.ApplyFuncs = append(cfg.ApplyFuncs, fixTODOComments())
+		applyFuncs = append(applyFuncs, fixTODOComments())
+	}
+
+	for _, fn := range cfg.ApplyFuncs {
+		applyFuncs = append(applyFuncs, fn)
```

Comment:
spread operator works here
```suggestion
	applyFuncs = append(applyFuncs, cfg.ApplyFuncs...)
```